test/daos/job.spec.js

```diff
@@ -116,4 +116,54 @@ describe('Dao: Job', function () {
       });
     });
   });
+  describe('updateJob', function () {
+    it('correctly calls the updateJob store method', async function () {
+      const mockJobStore = {
+        updateJob: sinon.stub().returns({
+          id: 1,
+          title: '1',
+          location: '2',
+          salary: '3',
+          job_type: '4',
+          summary: '5',
+          description: '6',
+          datePosted: '2020-01-01',
+        }),
+      };
+
+      const dao = new JobDao(mockJobStore);
+
+      const jobData = await dao.updateJob(
+        { id: 1 },
```

Comment:
This doesn't look correct. The `updateJob` method takes in a number as its first argument. Not an object.

Reply:
Does that mean the test I've put in is wrong as the test didn't return an error with the object there?